rust/headless-client/src/main.rs

```diff
@@ -175,6 +175,7 @@ fn main() -> Result<()> {
         Some(id) => id,
         None => device_id::get_or_create().context("Could not get `firezone_id` from CLI, could not read it from disk, could not generate it and save it to disk")?.id,
     };
+    firezone_telemetry::configure_scope(|scope| scope.set_tag("firezone_id", &firezone_id));
```

Comment:
The ID basically allows us to identify users right? Would we be getting any advantage from telling sentry that via `set_user`?

https://docs.rs/sentry/latest/sentry/struct.Scope.html#method.set_user

Reply:
It identifies the device, so I figured I'd make it a tag with a more specific name instead of using the well-known user ID field

Reply:
It would be cool to capture `/etc/machine-id` or a hash of that (I think the docs recommend not to use it directly), since that survives re-installs / file deletions